lighthouse/tests/beacon_node.rs

```diff
@@ -24,7 +24,9 @@ use types::non_zero_usize::new_non_zero_usize;
 use types::{Address, Checkpoint, Epoch, Hash256, MainnetEthSpec};
 use unused_port::{unused_tcp4_port, unused_tcp6_port, unused_udp4_port, unused_udp6_port};
 
-const DEFAULT_ETH1_ENDPOINT: &str = "http://localhost:8545/";
+const DEFAULT_EXECUTION_ENDPOINT: &str = "http://localhost:8545/";
```

Comment:
Slightly clearer conventional name:

```suggestion
const DEFAULT_EXECUTION_ENDPOINT: &str = "http://localhost:8551/";
```